File: languages/java/src/main/java/com/bloock/sdk/entity/identity_v2/BjjIdentityKey.java
package com.bloock.sdk.entity.identity_v2;

import com.bloock.sdk.bridge.proto.IdentityEntitiesV2;

/**
 * Represents an identity BJJ key used.
 */
public class BjjIdentityKey implements IdentityKey {
  IdentityKeyArgs args;

  /**
   * Creates a new BjjIdentityKey instance with the provided issuer key arguments.
   * @param args
   */
  public BjjIdentityKey(IdentityKeyArgs args) {
    this.args = args;
  }

  @Override
  public IdentityEntitiesV2.IdentityKey toProto() {
    IdentityEntitiesV2.IdentityKey.Builder builder = IdentityEntitiesV2.IdentityKey.newBuilder();

    if (this.args.localKey != null) {
      builder.setLocalKey(this.args.localKey.toProto());
    }

    if (this.args.managedKey != null) {
      builder.setManagedKey(this.args.managedKey.toProto());
    }

    return builder.build();
  }
}

File: languages/php/Bloock/Entity/IdentityV2/BjjIdentityKey.php
<?php

namespace Bloock\Entity\IdentityV2;

/**
 * Represents an identity BJJ key used.
 */
class BjjIdentityKey implements IdentityKey
{
    public IdentityKeyArgs $args;

    /**
     * Creates a new BjjIdentityKey instance with the provided issuer key arguments.
     * @param IdentityKeyArgs $args
     */
    public function __construct(IdentityKeyArgs $args)
    {
        $this->args = $args;
    }

    public function toProto(): \Bloock\IdentityKey
    {
        $s = new \Bloock\IdentityKey();

        if ($this->args->localKey != null) {
            $s->setLocalKey($this->args->localKey->toProto());
        }

        if ($this->args->managedKey != null) {
            $s->setManagedKey($this->args->managedKey->toProto());
        }

        return $s;
    }
}
